sys/arduino/base.cpp

```diff
@@ -62,6 +62,16 @@ void delay(unsigned long msec)
     xtimer_usleep(1000 * msec);
 }
 
+void delayMicroseconds(unsigned long usec)
+{
+    xtimer_usleep(usec);
+}
+
+unsigned int micros()
```

Comment:
Here the return type should be `unsigned long`.

Reply:
Many thanks for tests and spotting this! Corrected now, sorry was ok in the header but wrong here!

Thanks!
F.